identifiers/country-gb/country-gb.csv

```diff
@@ -0,0 +1,6 @@
+id,name,gss_code
+ocd-division/country:gb,United Kingdom,K02000001
+ocd-division/country:gb/part:eng,England,E92000001
+ocd-division/country:gb/part:nir,Northern Ireland,N92000002
+ocd-division/country:gb/part:sct,Scotland,S92000003
+ocd-division/country:gb/part:wls,Wales,W92000004
```

Comment:
Note that the top-level identifiers haven't been deleted (if they had, the compile script wouldn't run), just moved.. but if you want the file renamed, I can do.

Reply:
Oh I missed that, thanks for pointing out! They're fine here.